app/templates/components/ui-table/cell/cell-sponsor-sanitize.hbs

```diff
@@ -0,0 +1,3 @@
+{{#if (not-eq (get record column.propertyName) null)}}
```

Comment:
if a property is null, you don't need to compare it with null to make this if condition true. the `not-eq` is redundant.